scripts/limpar-prefixo-interno-fonte-dados.py

```
@@ -0,0 +1,164 @@
+        "resultado": "encontrado" if ok else "sem_achado_no_escopo",
+        "natureza": "escrita", "volume": ok,
+        "detalhe": (f"{ok} fichas tinham em fonte_dados uma entrada com prefixo 'interno:', "
+                    f"nota editorial que a view candidatos_publico ja escondia mas que a "
+                    f"tabela base expunha pela chave anon. Limpo na origem porque RLS e por "
+                    f"linha e revogar a coluna quebraria a view, que e security_invoker. "
+                    f"Reincidencia do que a migration 20260803142851 tratou. Achado do review "
+                    f"de seguranca de lancamento. Backup em backups/{bkp.name}"),
+        "executado_em": AGORA,
+    })
+    if st_log not in (200, 201):
+        raise SystemExit(f"ABORTA: escrita aplicada mas coleta_log falhou ({st_log}): {resp}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)